server/src/test/java/io/crate/auth/JwtAuthenticationIntegrationTest.java

```diff
@@ -163,6 +177,48 @@ public void test_can_authenticate_with_jwt_token() throws Exception {
             "WITH (jwt = {\"iss\" = '" + iss + "', \"username\" = '" + appUsername + "'})"
         );
 
+        HttpServerTransport httpTransport = cluster().getInstance(HttpServerTransport.class);
+        InetSocketAddress address = httpTransport.boundAddress().publishAddress().address();
+        URI uri = URI.create(String.format(Locale.ENGLISH, "http://%s:%s/", address.getHostName(), address.getPort()));
+
+        try (var client = HttpClient.newHttpClient()) {
+            HttpRequest request = HttpRequest.newBuilder(uri)
+                .header(HttpHeaderNames.AUTHORIZATION.toString(), "Bearer " + jwt)
+                .header(HttpHeaderNames.ORIGIN.toString(), "http://example.com")
+                .header(HttpHeaderNames.ACCESS_CONTROL_REQUEST_METHOD.toString(), "GET")
+                .build();
+            var resp = client.send(request, BodyHandlers.ofString());
+            assertThat(resp.body()).containsIgnoringWhitespaces("""
+                {
+                  "ok" : true,
+                  "status" : 200
+                  """);
+        }
```

Comment:
Why do we need this? We have dedicated `test_can_authenticate_with_jwt_token_cached` with special assertions